.github/CODEOWNERS

```diff
@@ -0,0 +1,42 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# Migrations must be approved in order to make sure the db export rake
+# task is still scrubbing sensitive info
+/lib/tasks/db/ @veganstraightedge
+/db/ @veganstraightedge
```

Comment:
this is the only "real" config in this file.

I left the other examples below just for the PR process, but will delete before merging

Reply:
Righteous. IMO, you should add yourself to this list too.